sw/host/ot_certs/tests/generic.hjson

```diff
@@ -0,0 +1,135 @@
+// Copyright lowRISC contributors.
```

Comment:
do you need another "generic.hjson" here for the test? can you reuse the one in sw/device/silicon_creator/lib/cert/generic.hjson ?

Reply:
The generic.hjson in `sw/device` will be remove to point to the one in `ot_certs`. I am also thinking about moving the example hsjon as well.